### PR TITLE
update the lbproxy post to use the seboolean for haproxy

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: _posts/2018-05-02-load-balanced-smart-proxies-with-haproxy.md [hunk 1]
```diff
@@ -96,7 +92,7 @@ As only the first Smart Proxy will host a PuppetCA, we point the Puppet agent to
 
 Please note that if you are using a different load balancer technology, you can use this configuration as reference.
 
-Configure SELinux to run in `permissive` mode.
+Configure SELinux to allow HAProxy to bind any port: `semanage boolean --modify --on haproxy_connect_an`
```
Comment (beav) [inline _posts/2018-05-02-load-balanced-smart-proxies-with-haproxy.md]: s/an/any/

Comment (evgeni) [reply _posts/2018-05-02-load-balanced-smart-proxies-with-haproxy.md]: good catch

**Comment by GregSutcliffe** [comment]
@beav good to merge then?

**Comment by beav** [comment]
@GregSutcliffe yup!

**Comment by GregSutcliffe** [comment]
@beav ha, I asked because I thought you needed someone to merge, didn't know you had commit. Carry on! :)

**Pull request merged**